glue/icons/icon_preview.html

```diff
@@ -47,6 +47,7 @@
     <td><img width="60" src='glue_contour.png'></td>
     <td><img width="60" src='glue_crosshair.png'></td>
     <td><img width="60" src='glue_lasso.png'></td>
+    <td><img width="60" src='glue_pixel_extraction.png'></td>
```

Comment:
anything pixel extraction related shouldn't be in this PR right? That code is still living in glue-solar for now?

Reply:
Sure, let me remove that from this PR